common/components/componentsBySection/CreateEvent/EventPreviewForm.jsx

```diff
@@ -26,7 +26,7 @@ class EventPreviewForm extends React.PureComponent<Props> {
       <React.Fragment>
         <DjangoCSRFToken />
         <input type="hidden" name="is_created" value="True" />
-        <AboutEventDisplay event={this.props.project} viewOnly={true} />
+        <AboutEventDisplay event={this.props.project} viewOnly={true} showHeaders={false}/>
```

Comment:
This currently will have no effect.  You will need to add a prop to AboutEventDisplay, and have it control the footer visibility here:
https://github.com/DemocracyLab/CivicTechExchange/blob/42fafaf998e6d6d0b97abb011c194efe84465a2f/common/components/componentsBySection/CreateEvent/AboutEventDisplay.jsx#L208

Reply:
@marlonkeating  I have created a new branch for this issue due to some merge conflicts, since the branch wa scretaed 5 months ago. the new pull request is under: https://github.com/DemocracyLab/CivicTechExchange/pull/951